db/migrate/20160426174031_create_companies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCompanies < ActiveRecord::Migration
+  def change
+    create_table :companies do |t|
+      t.string :name
```

Comment:
Have we captured a story to add the other company fields? (I think maybe we did for Company styling?)